srcpkgs/tiramisu/template

```diff
@@ -0,0 +1,20 @@
+# Template file for 'tiramisu'
+pkgname=tiramisu
+version=20211107
```

Comment:
```suggestion
version=2.0.20211107
```

Reply:
Oh right, forgot about that. I only used the workaround because originally, tiramisu had a dash in the tagged releases.

Reply:
The commit message should also be updated

Reply:
> The commit message should also be updated

ping @Vapourium